include/boost/fusion/container/vector/detail/config.hpp

```diff
@@ -15,6 +15,8 @@
   || defined(BOOST_NO_CXX11_RVALUE_REFERENCES) \
   || defined(BOOST_NO_CXX11_TEMPLATE_ALIASES)  \
   || defined(BOOST_NO_CXX11_DECLTYPE)) \
+  || defined(BOOST_NO_CXX11_FUNCTION_TEMPLATE_DEFAULT_ARGS) \
```

Comment:
`vector/detail/config.hpp` affects `fuson::vector`, it should be in [tuple/tuple_fwd.hpp](https://github.com/boostorg/fusion/blob/develop/include/boost/fusion/tuple/tuple_fwd.hpp#L14).